modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentShortcutResourceImpl.java

```diff
@@ -57,9 +85,26 @@ private DocumentShortcut _toDocumentShortcut(FileShortcut fileShortcut)
 				contextUser));
 	}
 
+	private List<DocumentShortcut> _toDocumentShortcuts(
+			List<DLFileShortcut> dlFileShortcuts)
+		throws Exception {
+
+		List<DocumentShortcut> documentShortcuts = new ArrayList<>();
+
+		for (DLFileShortcut dlFileShortcut : dlFileShortcuts) {
```

Comment:
@SamZiemer, I think we can use TransformUtil here (Brian will likely change this 😅).

Reply:
Hopefully he will be ok with this. I tried to stick to a similar case like we have [here](https://github.com/SamZiemer/liferay-portal/blob/master/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentResourceImpl.java#L933-L943) to keep it as alike other liferay code as I could.